.github/workflows/snyk-scan.yml

```diff
@@ -0,0 +1,67 @@
+name: Snyk Security Vulnerability Scan
+
+on:
+  pull_request:
+  push:
+    tags:
+      - 'RELEASE-[0-9]+.[0-9]+.[0-9]+.[0-9]+-[0-9]+'
+    branches:
+      - 'master'
+
+jobs:
+  snyk_scan_test:
+    if: ${{ github.event_name == 'pull_request' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0 # To fetch all commits history on branch (Refer: https://github.com/tj-actions/changed-files#usage)
+
+      - name: Check changed Deps files
+        uses: tj-actions/changed-files@v10
+        id: changed-files
+        with:
+          files: |
+            build.gradle$
+
+      - uses: snyk/actions/setup@master
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "8"
+          distribution: 'adopt'
+
+      - name: Snyk scan for Java dependancies
+        if: steps.changed-files.outputs.any_changed == 'true'
+        run: snyk test --all-projects -d --fail-on=all
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+
+  snyk_scan_monitor:
+    if: ${{ github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Extract github branch/tag name
+        shell: bash
+        run: echo "##[set-output name=ref;]$(echo ${GITHUB_REF##*/})"
+        id: extract_ref
+
+      - uses: snyk/actions/setup@master
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "8"
+          distribution: 'adopt'
+
+      - name: Snyk scan for Java dependancies
+        run: |
+          for file in $(find . -name "build.gradle"); do
+            file=$(echo $file | cut -c 3-)
```

Comment:
you could also just use ${file:2} below I think but that's just a detail

Reply:
Yes, it worked, thanks for pointing it out. I updated it